db/seeds.rb

```diff
@@ -1,7 +1,14 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'rspec'
+require 'spec_helper.rb'
+require 'rails_helper.rb'
+
+@service = Service.find_by url: 'localhost'
+unless @service
+  @service = FactoryGirl.create :service, name: 'dev stuff', url: 'localhost'
+  item_type = FactoryGirl.create :item_type, name: 'programmer', creator: @service
+  item = FactoryGirl.create :item, name: 'Tom', reservable: true, item_type: item_type
+  FactoryGirl.create :reservation, item: item, start_datetime: Time.parse('15/1/11').iso8601, 
```

Comment:
If these are supposed to be dates, I'd suggest Date.parse with a more standard format, e.g.:

```ruby
Date.parse('2016-04-22')
```